extensions/wikia/WikiaMaps/controllers/WikiaMapsMapController.class.php

```diff
@@ -11,7 +11,10 @@ class WikiaMapsMapController extends WikiaMapsBaseController {
 	 * @return Array
 	 */
 	public function getTileSets() {
-		$params = [];
+		$params = [
+			'sort' => 'desc',
+			'limit' => 49
```

Comment:
Oh, one more thing. We might want to add a comment line here why it is `49` - the thing about the upload button :) Extracting it to a const may be an overkill if it's being used only in this one place... on the other hand it's a good practice to keep "magic numbers" in const. I'd ask one or few more experienced developers what's their opinion about it :neckbeard: